modules/swagger-parser/src/main/java/io/swagger/parser/Swagger20Parser.java

```diff
@@ -129,6 +133,23 @@ private Swagger convertToSwagger(String data) throws IOException {
                 SwaggerDeserializationResult result = new SwaggerDeserializer().deserialize(rootNode);
 
                 Swagger convertValue = result.getSwagger();
+
+                if(convertValue.getPaths() == null) {
+                    // no need to validate Path Parameters since there are no paths
+                }
+                else {
+                    final Optional<Boolean> validPaths = allPathParametersAccountForExtractedPathParameters(convertValue);
+                    if(validPaths.isPresent()) {
+                        final Boolean validResult = validPaths.get();
+                        if(validResult.equals(Boolean.FALSE)) {
+                            return null;
```

Comment:
We can't abort the parse if there are parameters missing.  The `SwaggerDeserializer` is responsible for keeping track of messages and returning a potentially invalid spec if there are problems.